pkgs/applications/misc/electrum/grs.nix

```diff
@@ -63,8 +63,14 @@ python3.pkgs.buildPythonApplication {
     qdarkstyle
   ];
 
-  preBuild = ''
-    substituteInPlace ./electrum_grs/ecc_fast.py \
+  postPatch = ''
+    # make compatible with protobuf4 by easing dependencies ...
+    substituteInPlace ./contrib/requirements/requirements.txt \
+      --replace "protobuf>=3.12,<4" "protobuf>=3.12"
+    # ... and regenerating the paymentrequest_pb2.py file
+    protoc --python_out=. electrum_grs/paymentrequest.proto
+
+   substituteInPlace ./electrum_grs/ecc_fast.py \
```

Comment:
```suggestion
    substituteInPlace ./electrum_grs/ecc_fast.py \
```

Reply:
applied in 7c1eeef919affdd564a01ef0631891d5428eb8f2